src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -48,6 +48,9 @@
     <Reference Include="FSharp.Compiler.CodeDom">
       <HintPath>..\..\packages\FSharp.Compiler.CodeDom.1.0.0.1\lib\net40\FSharp.Compiler.CodeDom.dll</HintPath>
     </Reference>
+    <Reference Include="FSharp.Core">
+      <HintPath>..\..\packages\FSharp.Core.3.1.2.5\lib\net40\FSharp.Core.dll</HintPath>
```

Comment:
is 3.1.2.5 the right version? 4.3.4 is the current on [nuget.org](https://www.nuget.org/packages/FSharp.Core/)

Reply:
I need assembly version 4.3.1, and NuGet package 3.1.2.5 is the last one to provide that; laster NuGet packages provide version 4.4.0.

So we either use this NuGet package, or I figure out how to do binding redirects. :-/